test/bills.test.js

```diff
@@ -0,0 +1,69 @@
+const chai                          = require('chai');
+const chaiHttp                      = require('chai-http');
+const app                           = require('../server');
+const createUserAdminAndLogin       = require('./utils/createUserAdminAndLogin');
+const {createBills, deleteAllBills} = require('./utils/createBills');
+
+chai.use(chaiHttp);
+chai.should();
+
+const expect = chai.expect;
+
+let credentials;
+
+describe('Bills', () => {
+    beforeEach(async () => {
+        await deleteAllBills();
+        credentials = await createUserAdminAndLogin();
+    });
+
+    describe('POST /api/v2/bills', () => {
+        it('Create bill and get it with the code', async () => {
+            const bill = await createBills();
+            const res  = await chai.request(app)
+                .post('/api/v2/bills')
+                .set('authorization', credentials.token)
+                .send({PostBody: {filter: {code: bill.code}, limit: 99}});
+            expect(res).to.have.status(200);
+            expect(res.body.datas[0].name).be.equals(bill.name);
+        });
+        it('Create bill and get it with the id - w/o authentication', async () => {
```

Comment:
The description does not match your test
Should be something like this : `it('Should not retrieve bill if no authorization token send', ...`